### ISSUE TITLE
Convert the FIFE forums to something else

### ISSUE THREAD

**Issue by prock-fife** [fifengine/fifeweb#5]
Discourse is currently the front runner.  We need to find a place to host it though.

**Comment by emilhem** [comment]
Well, what are you using now? Is there any restrictions really? How will it be financed?

**Comment by prock-fife** [comment]
Unfortunately we have 0 financing. I have a request in with Discourse to host the forums for free and am waiting for a response.   Currently the forums are hosted on the same host as our website using SMF2 (php based forums).

**Comment by emilhem** [comment]
How is it financed? Is the current server free?
Also do you have access to the domain? I would suggest changing the TTL values to as little as possible to make the transition faster.

**Comment by prock-fife** [comment]
I currently donate the server/host.  I also have access to the domain to adjust TTL values.

**Comment by emilhem** [comment]
How much does it cost? A VPS at DigitalOcean can get as low as $5.

**Comment by prock-fife** [comment]
Yes it's not expensive to host.  But I would prefer free of course.

**Comment by LinuxDonald** [comment]
@prock-fife we could host the fife forum at the same place as we host the uh forum.

**Comment by jakoch** [comment]
We now have GitHub Discussions available: https://github.com/fifengine/fifengine/discussions
